src/lib/filter/adjudicators/defaultTestObjects.ts

```diff
@@ -0,0 +1,46 @@
+import { GroupVersionKind, kind, KubernetesObject } from "kubernetes-fluent-client";
+import { Event, Operation } from "../../enums";
+import { AdmissionRequest, Binding, Filters } from "../../types";
+
+export const defaultFilters: Filters = {
+  annotations: {},
+  deletionTimestamp: false,
+  labels: {},
+  name: "",
+  namespaces: [],
+  regexName: "^default$",
+  regexNamespaces: [] as string[],
+};
+
+const defaultGroupVersionKind: GroupVersionKind = {
+  kind: "some-kind",
+  group: "some-group",
+};
+
+export const defaultBinding: Binding = {
+  event: Event.ANY,
+  filters: defaultFilters,
+  kind: defaultGroupVersionKind,
+  model: kind.Pod,
+  isFinalize: false, //Lots of optionals that maybe don't belong here. Would be nice to choose to include
```

Comment:
This is related to my earlier comment about the `prettier-ignore` on and adjudicator